src/Sylius/Behat/Resources/config/services/api.xml

```diff
@@ -18,6 +18,11 @@
             <argument type="service" id="sylius.behat.shared_storage" />
         </service>
 
+        <service id="sylius.behat.api_platform_client.shop.address" class="Sylius\Behat\Client\ApiPlatformClient" parent="sylius.behat.api_platform_client">
+            <argument>address</argument>
```

Comment:
I'm not sure about this change, because in `UI` we have adressBook with path `addresses`, but generally we have  entity `Address` and in REST API we should create endpoint like `/address`, look in other entities and paths in UI and API

Reply:
check this `bin/console debug:router |grep new-api/admin/` ALL path in our API have resource name in plural form, and all services in `src/Sylius/Behat/Resources/config/services/api.xml` have an argument in the plural form. One of us is wrong and i believe that isn't me :D

Reply:
Yes you aright, I was looking wrong 🚀